src/Skoruba.IdentityServer4.Admin/Configuration/SendgridConfiguration.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Skoruba.IdentityServer4.Admin.Configuration
```

Comment:
This would be good to have in a shared project between both the Admin and the STS.Identity project.  This was pulled in from the STS.Identity project.